docs/sources/enterprise/saml.md

```diff
@@ -7,165 +7,126 @@ type = "docs"
-| `assertion_attribute_name`                                  | No       | Friendly name or name of the attribute within the SAML assertion to use as the user's name         | `displayName` |
-| `assertion_attribute_login`                                 | No       | Friendly name or name of the attribute within the SAML assertion to use as the user's login handle | `mail`        |
-| `assertion_attribute_email`                                 | No       | Friendly name or name of the attribute within the SAML assertion to use as the user's email        | `mail`        |
+| `metadata_valid_duration`                                   | No       | Duration, for how long the SP metadata is valid                                                  | `48h`         |
+| `assertion_attribute_name`                                  | No       | Friendly name or name of the attribute within the SAML assertion to use as the user name         | `displayName` |
+| `assertion_attribute_login`                                 | No       | Friendly name or name of the attribute within the SAML assertion to use as the user login handle | `mail`        |
+| `assertion_attribute_email`                                 | No       | Friendly name or name of the attribute within the SAML assertion to use as the user email        | `mail`        |
 
-### Cert and private key
+### Enable SAML authentication
+
+To use the SAML integration, set `enabled` to `true` in the Grafana configuration file, either `custom.ini` or `grafana.ini`. 
+
+Refer to [Configuration]({{< relref "../installation/configuration.md" >}}) for more information about configuring Grafana.
+
+### Certificate and private key
 
 The SAML SSO standard uses asymmetric encryption to exchange information between the SP (Grafana) and the IdP. To perform such encryption, you need a public part and a private part. In this case, the X.509 certificate provides the public part, while the private key provides the private part.
 
-Grafana supports two ways of specifying both the `certificate` and `private_key`. Without a suffix (e.g. `certificate=`), the configuration assumes you've supplied the base64-encoded file contents. However, if specified with the `_path` suffix (e.g. `certificate_path=`) Grafana will treat it as a file path and attempt to read the file from the file system.
+Grafana supports two ways of specifying both the `certificate` and `private_key`. 
+* Without a suffix (`certificate` or `private_key`), the configuration assumes you've supplied the base64-encoded file contents. 
+* With the `_path` suffix (`certificate_path` or `private_key_path`), then Grafana treats the value entered as a file path and attempt to read the file from the file system.
+
+You can only use one form of each configuration option. Using multiple forms, such as both `certificate` and `certificate_path`, results in an error.
```

Comment:
Does it? Or is there a quiet preference for either? I should look this up (unless this was moved here from somewhere and I missed it)

Reply:
This was buried in an "important note" directly above this heading: https://grafana.com/docs/grafana/latest/enterprise/saml/#grafana-configuration.

I moved it to where it would be more prominent and reworded it slightly for clarity.